src/App.tsx

```diff
@@ -1,9 +1,18 @@
-import React from 'react';
+import React, { useState } from 'react';
 import './App.scss';
 import { MoviesList } from './components/MoviesList';
 import moviesFromServer from './api/movies.json';
 
 export const App: React.FC = () => {
+  const [query, setQuery] = useState('');
+  const visibleMovies = moviesFromServer.filter((movie) => {
+    const { title, description } = movie;
+    const lowerQuery = query.toLowerCase().trim();
+
+    return description.toLowerCase().includes(lowerQuery)
+      || title.toLowerCase().includes(lowerQuery);
+  });
```

Comment:
You repeat the same logic twice `toLowerCase().includes(lowerQuery)`. Would be better to create a method for it